x-pack/platform/plugins/shared/fleet/server/services/preconfiguration/outputs.ts

```diff
@@ -212,8 +201,25 @@ async function hashSecrets(output: PreconfiguredOutput) {
         kibana_api_key: kibanaAPIKey,
       };
     }
+    if (typeof remoteESOutput.secrets?.ssl?.key === 'string') {
+      const key = await hashSecret(remoteESOutput.secrets?.ssl?.key);
+      return {
```

Comment:
I think the `ssl.key` should be added to the `secrets` variable that contains other secrets too, all of them should be hashed.
It seems this is incorrect in kafka type too in the existing logic.